src/phoenix/server/api/mutations/user_mutations.py

```diff
@@ -49,11 +56,11 @@ async def create_user(
         user_role_id = (
             select(models.UserRole.id).where(models.UserRole.name == role_name).scalar_subquery()
         )
-        secret = info.context.get_secret()
+        salt = secrets.token_bytes(DEFAULT_ENTROPY)
```

Comment:
does this mean we lose the ability to reset all users by changing the system secret?

Reply:
yes, it's a tradeoff: because the secret is also used to sign the jwt, we don't want one secret to be used for two different things